spec/html-spec.coffee

```diff
@@ -611,3 +611,26 @@ describe 'HTML grammar', ->
     it "tolerates hyphens in other tag names", ->
       lines = grammar.tokenizeLines '<foo-bar>'
       expect(lines[0][1]).toEqual value: 'foo-bar', scopes: ['text.html.basic', 'meta.tag.other.html', 'entity.name.tag.other.html']
+
+  describe "snippets", ->
+    snippetsModule = null
+
+    beforeEach ->
+      # FIXME: This should just be atom.packages.loadPackage('snippets'),
+      # but a bug in PackageManager::resolvePackagePath where it finds language-html's
+      # `snippets` directory before the actual package necessitates passing an absolute path
```

Comment:
How would you feel about including a link to the issue here? That way, future devs will have an easy way to find out whether the issue has been resolved, and that will let them know whether they can change the code to use `atom.packages.loadPackage('snippets')`.

Reply:
That reminds me - I still need to file an issue for it.

Reply:
Done!  See the line below.